### PR TITLE
Merging upstream changes

### PR DESCRIPTION
**Pull request by SJaffa**

### CODE REVIEW

**Comment by SJaffa** [comment]
TODO: 
- [ ] Sarah to run through the code and check it works on Myriad
- [ ] Check with Heather
- [ ] Read through all pages and check differences

**Review by dpshelio — APPROVED** [review]
I seen it works. Now it needs to check that the content is correct. Approving it to allow the merge when it's been checked.

**Comment by SJaffa** [comment]
@dpshelio do you know why the CircleCI failed here?

File: _episodes/11-connecting.md [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+computer we are logged onto can be checked with the `hostname` command. (You
+may also notice that the current hostname is also part of our prompt!)
+
+```
+{{ site.remote.prompt }} hostname
+```
+{: .language-bash}
+
+```
+{{ site.remote.host }}
```
Comment (SJaffa) [inline _episodes/11-connecting.md:360]: This doesn't give any output, should give `login12.myriad.ucl.ac.uk` or similar

File: _episodes/12-cluster.md [hunk 2]
```diff
@@ -1,237 +1,313 @@
...
-nodes.
+All interaction with the compute nodes is handled by a specialized piece of
+software called a scheduler (the scheduler used in this lesson is called
+{{ site.sched.name }}). We'll learn more about how to use the
+scheduler to submit jobs next, but for now, it can also tell us more
+information about the compute nodes.
 
-For example, we can view all of the worker nodes with the `{{ site.sched.info }}` command.
+For example, we can view all of the compute nodes by running the command
+`{{ site.sched.info }}`.
```
Comment (SJaffa) [inline _episodes/12-cluster.md:156]: `site.sched.info` should be `nodetypes` not `qhost`. qhost lists every node, nodetypes summarises the different types

File: _episodes/15-transferring-files.md [hunk 3]
```diff
@@ -1,253 +1,433 @@
...
 
-The syntax is: `wget https://some/link/to/a/file.tar.gz`. For example, download the lesson sample
-files using the following command:
+The syntax for these commands is: `curl -O https://some/link/to/a/file`
+and `wget https://some/link/to/a/file`. Try it out by downloading
+some material we'll use later on, from a terminal on your local machine.
 
 ```
-{{ site.remote.prompt }} wget {{ site.url }}{{ site.baseurl }}/files/bash-lesson.tar.gz
+{{ site.local.prompt }} curl -O {{ site.url }}{{ site.baseurl }}/files/hpc-intro-data.tar.gz
```
Comment (SJaffa) [inline _episodes/15-transferring-files.md:32]: `site.url` and `site.baseurl` are not specified

Comment (dpshelio) [reply _episodes/15-transferring-files.md:32]: They don't need to, there's a `repository` keyword that takes care of that when served on gh-pages.

File: _episodes/11-connecting.md (hunk 1 above)
Comment (dpshelio) [reply _episodes/11-connecting.md:360]: added the missing key

File: _episodes/12-cluster.md (hunk 2 above)
Comment (dpshelio) [reply _episodes/12-cluster.md:156]: Fixed to the right command.

**Pull request merged**